frontend/src/console/components/ConsolePage/__snapshots__/ConsolePage.spec.tsx.snap

```diff
@@ -12,34 +12,41 @@ exports[`Console Page Correctly renders loading page 1`] = `
     >
       <span
         className="status-pill badge badge-pill badge-primary"
-        onBlur={[Function]}
-        onFocus={[Function]}
-        onMouseOut={[Function]}
-        onMouseOver={[Function]}
-      >
-        <span
-          className="dot"
-          style={
-            Object {
-              "backgroundColor": "grey",
-            }
+        style={
+          Object {
+            "backgroundColor": "grey",
           }
-        />
+        }
+      >
         <div
           className="text"
         >
-          Status: loading
+          Loading
         </div>
+        <i
+          className="fas fa-question-circle fa-lg"
+          onBlur={[Function]}
+          onFocus={[Function]}
+          onMouseOut={[Function]}
+          onMouseOver={[Function]}
+        />
       </span>
       <button
-        className="float-right loading btn btn-primary btn-lg"
+        className="float-right disabledBtn btn btn-primary btn-lg"
```

Comment:
Although I'm not a reviewer here, just bumped into this PR, and I'd never ever want to ruin someone else's PR review, but could we stick with one class name style? We already has dash separated class names, and it would be a bit confusing/frustrating on the long run to have 2-3-4 different kind of class names 😇

Reply:
Hey, @gabor-boros  thanks for noticing that, it makes a lot of sense, @spokerman12  can you have a look and make all the styles consistent? i.e with dashed classNames.

Reply:
Thanks for spotting this, @gabor-boros! I have updated the classNames and their implications accordingly. I was already given this heads-up class name style but overlooked it on this particular PR; I'll make sure to apply this constantly.